plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraTransform.kt

```diff
@@ -62,4 +63,25 @@ internal object CameraTransform {
   }
 
   fun calculateScaleBy(amount: Double, currentZoom: Double) = log2(amount) + currentZoom
+
+  /**
+   * Extension function to wrap longitude to one world copy.
+   */
+  fun Point.wrapCoordinate(): Point {
+    val lng = wrap(this.longitude(), -LONGITUDE_MAX.toDouble(), LONGITUDE_MAX.toDouble())
+    return Point.fromLngLat(lng, this.latitude())
+  }
+
+  // Wrap value to the given range (including min, excluding max).
+  @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+  internal fun wrap(value: Double, min: Double, max: Double): Double {
+    if (value >= min && value < max) {
```

Comment:
should we compare all values with some eps? 
`1e-6` for example

Reply:
yes adding that :)